docs/docs-content/clusters/cluster-management/cluster-updates.md

```diff
@@ -140,3 +142,109 @@ You can follow these steps to validate all cluster update approaches.
 3. Select the cluster you updated, and navigate to the **Profile** tab of the cluster.
 
 4. Confirm that the cluster is now using an updated profile.
+
+## Palette Reconciliation Behavior
+
+When cluster profile updates are applied, Palette will apply the changes to the cluster but will not delete any
+resources that are no longer in use. This may be appropriate in some instances, but removing those resources will be
+necessary in others. This can be achieved by utilizing one of the two following methods.
+
+The first would be to utilize a clean deployment with a new cluster profile. The previous version could be deleted once
+the new cluster and cluster profile are deployed. The challenge, however, is when particular resources are in use.
+
+The second option is to use `kubectl` to scale the resource count to 0. While deleting the resource using something like
+K9s is possible, Palette will recreate it immediately.
+
```

Comment:
```suggestion
This section provides guidance on removing resources using the second method. 

```

Reply:
I suggest telling users which method we'll provide guidance on at the beginning.

You can then shorten the initial intro to the steps.